pages/install/conda-install.rst

```diff
@@ -16,24 +16,32 @@
       `Anaconda <https://www.anaconda.com/products/individual#Downloads>`__ and
       `Miniconda <https://docs.conda.io/en/latest/miniconda.html>`__ are Python
       distributions that include the ``conda`` package manager, which can be used to
-      install Cantera. Both distributions are available for Linux, macOS (Intel and
-      ARM/M1), and Windows. Note that installing Cantera using Conda will only provide
-      the Cantera :ref:`Python <sec-conda-python-interface>` or :ref:`MATLAB
-      <sec-conda-matlab-interface>` interfaces. If you want to use the other Cantera
-      interfaces (to use Cantera from Fortran, C++, or C) then see the :ref:`OS-specific
-      installation options <sec-install>`.
-
-Anaconda and Miniconda both include ``conda``; the difference is that Anaconda includes a large
-number of Python packages that are widely used in scientific applications, while Miniconda is a
-minimal distribution that only includes Python and Conda, although all of the packages available in
-Anaconda can be installed in Miniconda. For more details on how to use conda, see the `conda
+      install Cantera.
+
+Both the Anaconda and Miniconda distributions are available for Linux, macOS (Intel and
+ARM/M1), and Windows. Note that installing Cantera using Conda will only provide
+the Cantera :ref:`Python <sec-conda-python-interface>` or :ref:`MATLAB
+<sec-conda-matlab-interface>` interfaces. If you want to use the other Cantera
```

Comment:
If we can get `libcantera-devel` to properly compile on `conda` then we have headers and libraries also. These work well on Linux, and I also have some initial results on Windows.

Reply:
I think there's more to do as far as documenting `libcantera-devel`, so I've created a separate issue for that (#180).

Reply:
Sounds good. That's all i Have then!